package no.nav.hm.grunndata.register.exceptions

class ArticleNameAlreadyExistsOnSeriesException() : IllegalArgumentException("Article name already exists on series")
